tools/install.py

```diff
@@ -92,3 +93,9 @@ def run(ip_addr, hostname, config, scripts, user, toolchain, **kwargs):
 
     if config in ['ignore', 'i']:
         cprint('Ignoring configuration changes', 'blue', attrs=['bold'])
+        
+    # Pipe the git commit to file
+    versionFile = open("version.txt", "w")
```

Comment:
You should match your variable style with the rest of the file. This should be version_file